src/lara-plugin/plugins/embeddable-runtime-context.spec.ts

```diff
@@ -1,11 +1,10 @@
 import { generateEmbeddableRuntimeContext } from "./embeddable-runtime-context";
 import { IInteractiveState } from "../plugin-api";
 import { emitInteractiveAvailable, emitInteractiveSupportedFeatures } from "../events";
-// ACTIVITY_PLAYER_CODE:
```

Comment:
@mklewandowski can speak more to this, but as I recall the files in the `lara-plugin` folder were copied directly from LARA and the goal was to make as few changes as possible to prevent them from diverging from their LARA counterparts thus making future merges/synchronizations easier. These `ACTIVITY_PLAYER_CODE` and `LARA_CODE` comments indicated where code changes were required to support the activity player. All of which is just to say that changes made to these files should be made with some consideration to how/whether they will be synchronized with the corresponding code in LARA moving forward. Ultimately, a more robust form of code sharing (e.g. npm module) would clearly be preferable.

Reply:
This is exactly correct. The LARA plugin code was brought in in its entirety and we tried to mark with comments wherever code needed to diverge in hopes of making future code syncing easier.

Reply:
I would vote to not remove these comments. This was requested by @scytacki (unless he is ok removing them).

Reply:
I'm not sure if that's sustainable long-term. Plugin API is API types + implementation which is host-specific. For example, AP and LARA obtain class info in a very different ways. Or they download firestoreJWT using different methods. It applies to most of the functions provided by the Plugin API. But I don't remember exactly all the pieces of plugin-api. Maybe some things can be reused directly.

The big shared part is the types and interfaces. They're already in `@concord-consortium/plugin-api`, so one day we could try to remove AP copies of the types and use ones from the package.

Reply:
I'm fine with removing these comments now. I think the time has passed when it made sense to try to share the code between the two projects. We are getting much closer to shutting down the LARA runtime. And on top of this when I try to look at the comments now in order to figure out what the differences are it isn't helping me much. 

Sorry for the extra work these comments caused only to be thrown away.